js/everything.js

```diff
@@ -612,7 +618,7 @@ $(function() {
     $("#form").submit(function(event) {
         event.preventDefault();
         var formValue = $.trim($("#v").val());
-        var formValueTime = /&t=(\d*)$/g.exec(formValue);
+        var formValueTime = /[?&]t=(\d*)$/g.exec(formValue);
         if (formValueTime && formValueTime.length > 1) {
             formValueTime = parseInt(formValueTime[1], 10);
             formValue = formValue.replace(/&t=\d*$/g, "");
```

Comment:
Today youtube reference "Copy video URL at current time" looks like `https://youtu.be/hTWKbfoikeg?t=74`. By the way current version of regexp also can't manage querystrings like `?t=10&v=abcd` or `&t=10&v=abcd`. So, do you want to find some way for manual options setup? If so, what patterns are accepted as valid?